### DISCUSSION TITLE
Using Sidekick64 without wire

### DISCUSSION THREAD

**Discussion by odflor** [frntc/Sidekick64#81] [General]
Hi Pals:

What functions of the SideKick64 is disabled if I don't wire anything? Using only as a normal cartridge.

Best Regards

**Comment by frntc** [comment]
The CPU wire is only needed for kernal replacement. The SID-wire only if you want to emulate SIDs. Everything else works without wires (on a C64 -- on the VIC20 you need one for VIC-emulation).
